server/views/applications/pages/move-on/foreign-national.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "../layout.njk" %}
+
+{% block questions %}
+
+  <h1 class="govuk-heading-l">{{page.title}}</h1>
+
+  {% set fnInformed %}
+  {{ govukDateInput({
```

Comment:
We can / should use the `applyDateInput` macro here

Reply:
I didn't realise we had one, you thought of everything! Will add it in